pyglmnet/pyglmnet.py

```diff
@@ -561,6 +564,7 @@ def __init__(self, distr='poisson', alpha=0.5,
         self.random_state = random_state
         self.callback = callback
         self.verbose = verbose
+        self.show_progress = show_progress
```

Comment:
be careful adding new attributes and parameters to functions. They are easy to add but hard to remove :)